java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -124,6 +126,7 @@ public interface TemplateConstants {
   String JDBC_TO_SPANNER_JDBC_URL = "jdbctospanner.jdbc.url";
 
   String JDBC_TO_SPANNER_JDBC_DRIVER_CLASS_NAME = "jdbctospanner.jdbc.driver.class.name";
+  String JDBC_TO_SPANNER_JDBC_FETCH_SIZE = "jdbctospanner.jdbc.fetchsize";
```

Comment:
corresponding change in template.properties file is missing
Also please use a reasonable default fetch size if it makes sense (or zero) in template.properties

Reply:
I've added entries to this file. I've used default of zero. I chose zero so that each driver will choose it's own value. As far as I know only Oracle has an unsuitable default. I didn't want to force my own default on other JDBC drivers.

For my education: what is this file used for? I didn't have any failing tests in Jenkins without this edit.

Reply:
I found where the file is used. Is my understanding correct that it is used to provide default values?

Reply:
Historically we were using a different cli parser, which forced us to have cli parameter name declared in template.properties
Down the line it was changed to current parser, which has removed that restriction.
Currently we are still following that convention. 
Which I think is ok, but if we want to change I am open to discuss it.